.travis.yml

```diff
@@ -11,14 +11,13 @@ cache:
     - $HOME/google-cloud-sdk
     - $HOME/.m2
 script:
-  - ./mvnw -T 1.5C test -P it -B ${INTEGRATION_TEST_FLAGS}
+  - ./mvnw -T 1.5C test -B ${INTEGRATION_TEST_FLAGS}
 install:
-  - ./mvnw -T 1.5C install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+  - ./mvnw -T 1.5C install -P it -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
But we have here `-DskipTests=true`. Doesn't that mean that IT tests won't go as well?

Reply:
I suspect that the tests aren't run, but it is here that the test structure is built.
i.e., it is here that what we're doing in https://github.com/spring-cloud/spring-cloud-gcp/pull/636/files#diff-2d9cc6b52cccb7c9ee54209384519466 kicks in

At least, I tried to set it in the test and it didn't work. Let's see if this works...

Reply:
Oof. I think I finally got it :)
Now Jenkins should also work because we're not using the `it` profile there (`unit` is used by default), and the `STACKDRIVER` appender is never included.

Reply:
Does it mean it's ready for the final review?
Thanks